src/community/china.md

```diff
@@ -6,74 +6,171 @@ toc: true
 
 {% assign path = 'flutter_infra_release/releases/stable/windows/flutter_windows_3.3.0-stable.zip' -%}
 
-The Flutter community has made a Simplified Chinese version
-of the Flutter website available at
-[https://flutter.cn](https://flutter.cn),
-maintained by [The China Flutter User Group (CFUG)][].
+To speed the download and installation of Flutter in China,
+consider using a mirror site.
 
-If you'd like to install Flutter using an 
-[installation bundle]({{site.url}}/release/archive),
-you can replace the domain of the original URL with a trusted mirror
-to speed it up. For example:
+{{site.alert.important}}
+  Use mirror sites only if you _trust_ the provider.
+  The Flutter team cannot verify their reliability or security.
+{{site.alert.end}}
+
+## Use a Flutter mirror site
+
+The [China Flutter User Group][] (CFUG) maintains a Simplified Chinese
+Flutter website [https://flutter.cn](https://flutter.cn) and a mirror.
+Other mirrors can be found at the [end of this guide](#known-trusted-community-run-mirror-sites).
+
+### Configure your machine to use a mirror site
+
+To install or use Flutter in China,
+use a trustworthy Flutter mirror site.
+This requires setting two environment variables on your machine.
+
+_All examples that follow presume that you are using the CFUG mirror._
+
+To set your machine to use a mirror site on macOS or Linux:
+
+1. Create and change into the directory where you plan
+   to store your local Flutter clone.
+
+1. Set your pub URL to your mirror site.
+
+   ```terminal
+   $ export PUB_HOSTED_URL=https://pub.flutter-io.cn
+   ```
+
+1. Set your Flutter storage URL to your mirror site.
+
+   ```terminal
+   $ export FLUTTER_STORAGE_BASE_URL=https://storage.flutter-io.cn
+   ```
+
+1. Clone the Flutter repo from your mirror site.
+
+   ```terminal
+   $ git clone -b dev {{site.repo.flutter}}.git
+   ```
+
+1. Add Flutter to your `PATH` environment variable.
+
+   ```terminal
+   $ export PATH="$PWD/flutter/bin:$PATH"
+   ```
+
+1. Run Flutter Doctor to verify your installation.
+
+   ```terminal
+   $ flutter doctor
+   ```
+
+1. Return to the [setting up Flutter]({{site.url}}/get-started/editor)
+   guide and continue from that procedure.
+
+From this example, `flutter pub get` fetches packages from `flutter-io.cn`.
+in any terminal where you set `PUB_HOSTED_URL` and `FLUTTER_STORAGE_BASE_URL`.
+
+Any environment variables set using `export` in this procedure
+only apply to the current terminal window.
+To set these values on a permanent basis,
```

Comment:
Windows tutorial (didn't find an official one):
https://docs.oracle.com/en/database/oracle/machine-learning/oml4r/1.5.1/oread/creating-and-modifying-environment-variables-on-windows.html#GUID-DD6F9982-60D5-48F6-8270-A27EC53807D0